examples/azuread/README.md

```diff
@@ -0,0 +1,29 @@
+# Social Auth Pipeline AzureAD Group Sync Example
+
+This example shows how to extend the Social Auth Pipeline to read groups from a groups claim in AzureAD and sync those with Nautobot.  
+
+Create a python module with the `group_sync.py` file in it, this could be done as part of a plugin or as a standalone python module.
+
+In the nautobot_config.py set the following values with the settings from Azure:
```

Comment:
Don't you also need to set `social_core.backends.azuread.AzureADOAuth2` in `AUTHENTICATION_BACKENDS`?

Reply:
You're right! I've added that to the README